Developing features

uimarshall opened the pull request:
This project entails the cloning of the Newsweek web using bootstrap. 
In this project I did the following:

- Layout the structure of the page
- Styled the page with the bootstrap framework with minimal css.
- Design for different screen sizes.

**After Review**

- Made sure that the text of my links always stays in one line
- Remove !important from css  for best practices
- Validate code in W3C
**New Changes After Review**
- Made sure that the navbar links is taking  in one line
- Used the recommended stickler configuration
- Removed commented code in css.

uimarshall committed:
Add the index file and content for headers and nav

uimarshall committed:
Add navbar items and articles section

uimarshall committed:
Add and style the debate section

uimarshall committed:
Add stickler and reset css file

uimarshall committed:
Add opinion section

uimarshall committed:
Add content to opinion section and styles

uimarshall committed:
Add and styled heroes of pandemic section

uimarshall committed:
Add and styled more stories section

uimarshall committed:
Add and style magazine section

uimarshall committed:
Implement responsiveness

uimarshall committed:
Refactored featured stories section

uimarshall committed:
Add mobile screen sizes to in-the-magazine section

uimarshall committed:
Add and styled the editors section

uimarshall committed:
Add the section for US in varities section

uimarshall committed:
Add and style World section in varieties section

uimarshall committed:
Add business and Tech section

uimarshall committed:
Add and style culture section

uimarshall committed:
Add and style sport section

uimarshall committed:
Add and style subscription section

uimarshall committed:
Add footer structure

uimarshall committed:
Add Styles to the footer

uimarshall committed:
Styled the header

uimarshall committed:
Add responsiveness for different screen sizes

uimarshall committed:
Refactor code, fix linting and update README

uimarshall committed:
Fix linting errors and update README file

andresporras commented on css/style.css:
```diff
@@ -0,0 +1,495 @@
+/* HEADER */
+
+.header-wrapper {
+  background-color: #f72210 !important;
```
It's not a good practice to use !important

andresporras commented on index.html:
```diff
@@ -0,0 +1,2466 @@
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Replace this line with:
`<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">`

andresporras commented on index.html:
```diff
@@ -0,0 +1,2466 @@
+							<a class="nav-link pt-0 border-right font-weight-bold"
+								>Tech &amp; Science</a
+							>
+						</li>
```
Make sure the text of your links always stays in one line (like in the original page)

andresporras commented:
### **Pull Request Under Review By Oscar Russi (TSE)**

andresporras reviewed (changes requested):
### Status: Change Required :recycle:
**Hello @uimarshall** 
Awesome job, you make great use of bootstrap classes, your page looks really close to the original  :clap:
There are still some improvements to do:

- [ ]  Check HTML markup with this validator https://validator.w3.org/#validate_by_upload By now the validator shows some errors in your markup.
- [ ] Don't use !important in your CSS
- [ ] Make sure the text of navbar links stay always in one line, like in the original page

Check the inline comments I did in some parts of your code 

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

uimarshall committed:
Fix code after review

The nav-links was fixed from overflowing
remove !important value in css
validate html in w3c.

HADDADSOHAIB commented on css/style.css:
```diff
@@ -0,0 +1,507 @@
+  width: 29px;
+  height: 28px;
+
+  /* margin-right: 10px; */
```
This should be removed

HADDADSOHAIB reviewed (changes requested):
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @uimarshall    :handshake: 

### Good job so far :clap:  your project is looking good :medal_sports: 

### there are some changes to get the project approved :point_down:
- The nav bar still taking two lines:
Your project:
![Screenshot from 2020-06-05 11-02-02](https://user-images.githubusercontent.com/26064042/83864198-524e2f80-a71c-11ea-98a7-4a2e169fe31f.png)
Original:
![Screenshot from 2020-06-05 11-02-18](https://user-images.githubusercontent.com/26064042/83864226-5b3f0100-a71c-11ea-83b5-9658d22c2ee1.png)
Please try to fix that, it should look like the original.
- I think you modified your stickler configuration, please use this file and don't change it:
https://github.com/microverseinc/linters-config/blob/Stickler/css/stylelint.config.js
- There commented code in your CSS, please remove that:
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
### After you are done with the changes, ask for another code review, Good Luck :+1:

uimarshall commented:
> > ### Pull Request under the review by Sohaib Haddad. (TSE)
> 
> ### Change Required ♻️
> ### Hello @uimarshall 🤝
> ### Good job so far 👏 your project is looking good 🏅
> ### there are some changes to get the project approved 👇
> * The nav bar still taking two lines:
>   Your project:
>   ![Screenshot from 2020-06-05 11-02-02](https://user-images.githubusercontent.com/26064042/83864198-524e2f80-a71c-11ea-98a7-4a2e169fe31f.png)
>   Original:
>   ![Screenshot from 2020-06-05 11-02-18](https://user-images.githubusercontent.com/26064042/83864226-5b3f0100-a71c-11ea-83b5-9658d22c2ee1.png)
>   Please try to fix that, it should look like the original.
> * I think you modified your stickler configuration, please use this file and don't change it:
>   https://github.com/microverseinc/linters-config/blob/Stickler/css/stylelint.config.js
> * There commented code in your CSS, please remove that:
> * [ ]  Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
> 
> ### After you are done with the changes, ask for another code review, Good Luck 👍
@HADDADSOHAIB I did fix the navbar, I think I did not update it on the live demo.
I will update now.

uimarshall committed:
Refactor code to reflect changes after review

uimarshall committed:
Update README.md

andresporras3423 commented:
### **Pull Request Under Review By Oscar Russi (TSE)**

andresporras3423 reviewed (approved):
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello @uimarshall** 
Amazing job :clap: :clap: you have met all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 
Don't forget to merge with the master branch

Keep going like this,

Oscar Andrés Russi Porras
